rpcs3/rpcs3.vcxproj

```diff
@@ -89,7 +89,7 @@
       <ExternalWarningLevel>TurnOffAllWarnings</ExternalWarningLevel>
     </ClCompile>
     <Link>
-      <AdditionalDependencies>DbgHelp.lib;Ole32.lib;gdi32.lib;..\hidapi.lib;..\libusb-1.0.lib;winmm.lib;miniupnpc_static.lib;rtmidi.lib;imm32.lib;ksuser.lib;version.lib;OpenAL32.lib;XAudio.lib;GLGSRender.lib;shlwapi.lib;VKGSRender.lib;vulkan-1.lib;wolfssl.lib;libcurl.lib;Wldap32.lib;glslang.lib;OSDependent.lib;OGLCompiler.lib;SPIRV.lib;MachineIndependent.lib;GenericCodeGen.lib;Advapi32.lib;user32.lib;zlib.lib;..\libpng16.lib;asmjit.lib;yaml-cpp.lib;discord-rpc.lib;emucore.lib;dxgi.lib;shell32.lib;Qt6Core.lib;Qt6Gui.lib;Qt6Widgets.lib;Qt6Concurrent.lib;Qt6Core5Compat.lib;Qt6Multimedia.lib;Qt6MultimediaWidgets.lib;Qt6Svg.lib;Qt6SvgWidgets.lib;7zlib.lib;SPIRV-Tools.lib;SPIRV-Tools-opt.lib;libcubeb.lib;cubeb.lib;soundtouch.lib;Avrt.lib;SDL.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
why are you removing this?
And if it is valid, why are you keeping all the other references?

Reply:
Also, this has nothing to do with the other changes and is totally out of place here.

Reply:
I removed `Qt6Core5Compat.lib` because VS could not find it when I tried building it, but now I see that I have Qt 6.5.2 installed and not Qt 6.6.1 like it says in `BUILDING.md`. I will try updating my Qt and compiling again.

Reply:
It's an extra module you have to install and is unrelated to the specific Qt 6 version.
It's possible that it is obsolete, but I don't think this should be part of this PR.

Reply:
I reverted this change. Thank you, I didn't know that it's a separated module. I have installed it now.